gwpy/timeseries/timeseries.py

```diff
@@ -1428,14 +1428,15 @@ def taper(self, side='leftright'):
         out *= planck(out.size, nleft=nleft, nright=nright)
         return out
 
-    def whiten(self, fftlength, overlap=0, method='scipy-welch',
-               window='hanning', detrend='constant', asd=None, **kwargs):
-        """White this `TimeSeries` against its own ASD
+    def whiten(self, fftlength=2, overlap=0, method='scipy-welch',
```

Comment:
why was a default added here? and why this value?

Reply:
I added the default because under the new scheme, `fftlength` isn't used at all if `fir=True` and `asd` is not `None`. I chose a default of 2 seconds because that's the smallest use case I see for this parameter, and I see it quite often.